Detectors/MUON/MCH/Raw/ElecMap/src/FeeLinkId.cxx

```diff
@@ -31,7 +31,7 @@ FeeLinkId decodeFeeLinkId(uint32_t x)
 {
   uint16_t feeId = static_cast<uint16_t>((x & 0xFFFF0000) >> 16);
   uint16_t linkId = static_cast<uint8_t>(x & 0xF);
-  return FeeLinkId(feeId, linkId);
+  return {feeId, linkId};
```

Comment:
This code produces a compiler error on Mac builder:

```
/System/Volumes/Data/build/ci_checks/alice_ci_0/o2/sw/SOURCES/O2/7714/0/Detectors/MUON/MCH/Raw/ElecMap/src/FeeLinkId.cxx:34:18: error: non-constant-expression cannot be narrowed from type 'uint16_t' (aka 'unsigned short') to 'uint8_t' (aka 'unsigned char') in initializer list [-Wc++11-narrowing]
  return {feeId, linkId};
                 ^~~~~~
```
The` linkId` is declared as `uint16_t` while it takes value of  `uint8_t`, what is also expected in the class constructor.